### DISCUSSION TITLE
Use the [[ operator for extracting a single element from an object

### DISCUSSION THREAD

**Discussion by PMassicotte** [R-nvim/R.nvim#156] [Ideas]
Think that could fit?

![Peek 2024-06-24 17-06](https://github.com/R-nvim/R.nvim/assets/4519221/a7d3b479-1305-464d-9b86-116f65dc26a1)

**Comment by PMassicotte** [comment]
It is usually safer to use `[[` because of partial matching:

```r
mtcars$mp
mtcars[["mp"]]
mtcars[["mpg"]]
```

**Comment by jalvesaq** [comment]
Yes, I see it as a good new feature because it changes the code to fix another error pointed out by `lintr`.
